### PR TITLE
Advisory Board Meeting Minutes 2019-05-15

### PR DESCRIPTION
**Pull request by karmencj**
Posting Advisory Board Meeting Minutes 2019-05-15

image

### CODE REVIEW

File: content/datasets/_index.md [hunk 1]
```diff
@@ -9,7 +9,9 @@ The Open Force Field Initiative makes a number of resources available to the com
 
 ---
 
-- [open-forcefield-data](https://github.com/openforcefield/open-forcefield-data): Curated datasets from the NIST ThermoML database to be used for force field parameterization.
+- [open-forcefield-data](https://github.com/openforcefield/open-forcefield-data): 
+  * Curated datasets from the NIST ThermoML database to be used for force field parameterization:
+    +
```
Comment (j-wags) [inline content/datasets/_index.md]: This extra space is rendering as a blank bullet point. You should remove it (one of the small joys of markdown)!

File: content/consortium/minutes/2019-05-15-minutes.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+weight: 10
+author: "Karmen Condic-Jurkic and Arjun Narayanan"
+---
+
+The [Open Force Field Consortium Advisory Board](https://openforcefield.org/consortium/) met on 15 May, 2019.
+The minutes are summarized as follows:
+
+### Open Forcefield Advisory Board Meeting Minutes – May 15, 2019
+
+2019/05/15 Advisory Board Meeting Minutes
```
Comment (j-wags) [inline content/consortium/minutes/2019-05-15-minutes.md]: This line is redundant.

File: content/consortium/minutes/2019-05-15-minutes.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+- A subset of 820 1D torsions was selected for initial scanning, but this set will be expanded to include more torsion and 2D scans, which are currently omitted due to computational cost;
+
+- Current level of theory used to compute torsion profiles is B3LYP-D3(BJ)/DZVP, but a subset of 15 torsions has been selected for further QM benchmarking (more details on datasets and benchmarking can be found in this [Science Update](https://openforcefield.org/science/updates/2019-05-16-condicj/));
+
+- [ForceBalance](https://github.com/leeping/forcebalance) is ready to compute torsion parameters.
+
+**Valence and impropers update** (D. Mobley):
+
+- Some initial refitting on of impropers on trivalent nitrogens using a small set of molecules with different substituents on nitrogen has been done based on SMIRNOFF99Frosst parameters, but more molecules will be added to this dataset (pulled from eMolecule with fingerprint addition);
```
Comment (j-wags) [inline content/consortium/minutes/2019-05-15-minutes.md]: Some initial refitting ~on~ of impropers

File: content/datasets/_index.md (hunk 1 above)
Comment (karmencj) [reply content/datasets/_index.md]: Oh, I that's probably the extra space I added in my attempt to get bullet points to work in the first place. Thanks for catching it!

File: content/consortium/minutes/2019-05-15-minutes.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+- Impropers are being scanned while keeping torsions fixed using [geomeTRIC](https://github.com/leeping/geomeTRIC)- simplified and independent fitting runs will be performed initially to test the framework, but ultimately, ForceBalance should be used for fitting with all data simultaneously;
+
+- Data generated for torsions might be used for an initial valence fitting round;
+
+- C. Bannan is working on porting [Chemper](https://github.com/MobleyLab/chemper) to AMBER (work funded by MolSSI)
```
Comment (j-wags) [inline content/consortium/minutes/2019-05-15-minutes.md]: @bannanc -- I think something may have been lost in live-note-taking-shorthand here, but I'm not up to date with your current efforts. Could you confirm this, or provide a better sentence? (sorry for the ping out of the blue)

File: content/consortium/minutes/2019-05-15-minutes.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+
+**Logistics:**
+
+- Election for the Advisory Board- nominations to be sent to Karmen and David;
+
+- Renewal for Year 2- CRA needs to be signed by Sep 1. Industry partners need to check what is necessary for continuation and OFFI needs to provide necessary information and materials;
+
+- Involvement in the subgroup meetings - a visible schedule to make it easier for interested parties to join. In the meantime, contact Karmen to provide more information;
+
+- Better-ccessibility of slides and data-- Zenodo Community about to be formed to address this problem.
```
Comment (j-wags) [inline content/consortium/minutes/2019-05-15-minutes.md]: Better _a_cessibility

**Comment by karmencj** [comment]
@bannanc, would the following phrase be correct:

_C. Bannan is working on using Chemper to port AMBER force field parameters into SMIRNOFF format._

File: content/consortium/minutes/2019-05-15-minutes.md (hunk 4 above)
Comment (bannanc) [reply content/consortium/minutes/2019-05-15-minutes.md]: Here's an edit of what Karmen wrote below:
C. Bannan is working on using Chemper to port AMBER protein force field parameters into SMIRNOFF format.

Comment (j-wags) [reply content/consortium/minutes/2019-05-15-minutes.md]: Thanks, Caitlin!

**Comment by j-wags** [comment]
I've approved merging once Caitlin's suggested wording goes in.

File: content/consortium/minutes/2019-05-15-minutes.md (hunk 4 above)
Comment (karmencj) [reply content/consortium/minutes/2019-05-15-minutes.md]: Thanks both!

**Pull request merged**